src/containers/MainPage/MainPage.js

```diff
@@ -33,15 +31,19 @@ class MainPage extends Component {
   onSubmit = () => {
     const { query } = this.state;
     const { history } = this.props;
-    if (query === '' || query === undefined) {
```

Comment:
Instead of trying to use many negative conditions you can directly use ```if (!query) {}```, it will handle all the cases of empty string, undefined, null or 0

Reply:
Oh yeah. :innocent: